internal/node/controller/init.go

```diff
@@ -41,12 +41,12 @@ func generatePDSAppConfig(nodeConfig *config.NodeConfig) types.PostAppRequest {
 							Target: "/pds",
 						},
 					},
-					"exposed_ports": []string{"5000"},
+					"exposed_ports": []string{"5001"},
```

Comment:
was this wrong before?

Reply:
This one is fun. On MacOS, AirPlay will occasionally listen on port 5000, making this break. So apparently, never use port 5000 https://forums.developer.apple.com/forums/thread/682332